src/@next/components/containers/I18nLoader/context.ts

```diff
@@ -2,7 +2,6 @@ import { createContext } from "react";
 import { II18nContext } from "./types";
 
 export const I18nContext = createContext<II18nContext>({
-  catalogs: {},
   language: "en",
   languages: { en: "English" },
   setLanguage: () => undefined,
```

Comment:
> () => undefined

`(lang: string) => void`

Reply:
We have a default value here, in a separate file we keep the type definition.